Dockerfile

```diff
@@ -2,12 +2,10 @@ FROM php:5.6-apache
 
 # docker php image has its own way of installing a module
 RUN apt-get update \
- && apt-get install --no-install-recommends -y libpq-dev=9.6.23-0+deb9u1 \
+ && apt-get install --no-install-recommends -y libpq-dev=10.19-0ubuntu0.18.04.1 \
```

Comment:
This (new) is also triggering an error:

```
#5 7.909 E: Version '10.19-0ubuntu0.18.04.1' for 'libpq-dev' was not found
```

Reply:
Yeah... I tried at first the version 9.6.23-0+deb9u1 found on [this page](https://packages.debian.org/fr/stretch/libpq-dev).
Since it didn't worked I've then sshed into the VM and from an `apt-get update  && apt-get install --no-install-recommends -y libpq-dev` I retrieved this version, which is still failing in the last build of this PR.
I don't know (yet?) what version number we should use here.

Reply:
Don't forget that it's a container: you have to build it somewhere and exec in it, to then check which version is availabel, as it depends on which OS and distribution it is (which is different than a Cloud VM with Ubuntu).

Reply:
Doh... I didn't though about sshing in the Docker container on the VM to check the available version... `9.6.24-0+deb9u1` is the last one available when no version is specified.